bundles/org.openhab.binding.miio/src/main/resources/OH-INF/thing/commonChannels.xml

```diff
@@ -217,10 +221,15 @@
 		<label>Temperature</label>
 		<state readOnly="true" pattern="%.1f %unit%"/>
 	</channel-type>
+	<channel-type id="temperatureCelsius">
```

Comment:
Please use quantitype Number:Temperature
Unless you need something specific, the existing channeltype temperatureC can be re-used.

Reply:
sorry, not sure how I missed the existing channeltype, I am going to reuse it.